assets/scripts/fe/src/app/components/projects/projects.states.ts

```diff
@@ -10,5 +13,17 @@ export const PROJECTS_STATES : Object[] = [
     url     : '/projects/',
     views   : NavContent(ProjectsComponent),
     onEnter : LoginRequired
-  }
+  },
+  {
+    name    : 'project-details',
+    url     : '/project/:id/',
+    views   : NavContent(ProjectDetailsComponent),
+    onEnter : LoginRequired
+  },
+  {
+    name    : 'project-details-report',
```

Comment:
this should be merged with the one on dashboard so that the URL are the same. for now i'll let this pass as i should be the one to figure this out. good job